.github/workflows/dockers.yml

```diff
@@ -35,3 +36,33 @@ jobs:
 
       - name: Push Docker image to Docker Hub
         run: docker push ${{ secrets.DOCKERHUB_USERNAME }}/xtensa-esp32-c:latest
+
+  build-risc-v-esp32c6-c-dependencies-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Login to Docker Hub
+        run: echo "${{ secrets.DOCKERHUB_TOKEN }}" | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
+
+      - name: Build Docker image with risc-v esp32c6 C dependencies
+        run: docker build -f ./dockers/risc-v-esp32c6/c-dependencies/risc-v-esp32c6-c.dockerfile -t ${{ secrets.DOCKERHUB_USERNAME }}/risc-v-esp32c6-c .
+
+      - name: Push Docker image to Docker Hub
+        run: docker push ${{ secrets.DOCKERHUB_USERNAME }}/risc-v-esp32c6-c:latest
+
+  build-esp-c-dependencies-docker:
```

Comment:
What's the purpose of this docker?

Reply:
This is a contemporary solution for running CI on future, not yet implemented platforms
e.g. developer writes his own example for esp32c3 and he can test it immediately, just by using this docker image containing all esp platforms! There is no reason to going into CI and writing `build-risc-v-esp32c3-c-dependencies-docker`, then open pr etc...
Of course, using this image will take longer than using an image only for esp32c3, but it's a fast solution without needs to create separate pr or ask you to upload docker image with a specific architecture